src/main/java/io/stargate/sgv2/jsonapi/service/resolver/model/impl/CreateCollectionCommandResolver.java

```diff
@@ -419,7 +422,7 @@ private void validateUserParameters(
             userParamName -> {
               if (!expectedParamNames.contains(userParamName)) {
                 throw ErrorCode.INVALID_CREATE_COLLECTION_OPTIONS.toApiException(
-                    "Unexpected parameter '%s' for the provider '%s' provided",
+                    "Unexpected parameter '%s' for the provider '%s' provider",
```

Comment:
I think the original was correct, change is wrong :)

Reply:
oh, my bad